src/pages/invoice.js

```diff
@@ -38,7 +38,7 @@ class InvoicePage extends React.Component {
       Letter: {
         unit: 'in',
         page: {
-          width: 8.5,
+          width: 7.5,
```

Comment:
This is a hack that will create confusion.
The proper dimensions of the Letter format is 8.5 x 11 https://en.wikipedia.org/wiki/Letter_%28paper_size%29
Let's respect conventions here. You can play with margins, paddings, etc if needed.